horovod/spark/torch/remote.py

```diff
@@ -36,9 +36,12 @@
 
 
 def RemoteTrainer(estimator, metadata, last_checkpoint_state, run_id, dataset_idx):
+    # Utility functions
+    make_list_if_not_list = make_list_if_not_list_fn()
```

Comment:
Since we're using this function before entering the remote code, no need to make it a generator function.

Reply:
fixed.